src/connectors/dispatcher/impl.js

```diff
@@ -43,5 +45,65 @@ module.exports = new class extends Connector {
         return result;
     }
 
+    async fetchPlaybookRuns (filter = null, fields = null, sort_by = null) {
+        const uri = new URI(config.platformHostname);
+        uri.path('/api/playbook-dispatcher/v1/runs');
```

Comment:
Does this behave differently than uri.segment() used in the postPlaybookRunRequests function?

Reply:
Hey!  Yeah so, IIRC uri.segment() adds an additional `/` infront and behind whatever is passed to it, so I could have done something like:
> uri.segment('api');
> uri.segment('playbook-dispatcher')';
...etc

but I figured this would look cleaner.